src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionPatcher.java

```diff
@@ -55,6 +72,15 @@ private DissolutionDirector findDirector(String email, Dissolution dissolution)
                 .get();
     }
 
+    private void addPaymentInformation(String paymentReference, Timestamp paidAt, Dissolution dissolution) {
+        final PaymentInformation information = new PaymentInformation();
```

Comment:
All object creation should be done in mappers to make the service easier to test.